deploy/values.yaml

```diff
@@ -105,7 +105,7 @@ application:
     annotations:
       sealedsecrets.bitnami.com/cluster-wide: "true"
     files:
-      - name: nordmart-review-slack-webhook
+      slack-webhook:
```

Comment:
@faizanahmad055 will it be **slack-webhook** or **review-slack-webhook**

Reply:
slack-webhook is the postfix and the complete name will be review-slack-webhook.